server/src/controllers/order.controller.ts

```diff
@@ -54,39 +57,53 @@ const OrderController = {
     } catch (error) {
       return res.status(500).json({ message: error.message });
     }
+  },
+
+  async payAmount(req: Request, res: Response): Promise<Response> {
+    let isPaymentSucess = "1";
```

Comment:
A boolean can be used to track isPaymentSuccess

Reply:
Done